products/rhel8/profiles/stig.profile

```diff
@@ -435,6 +435,7 @@ selections:
     - dir_perms_world_writable_root_owned
 
     # RHEL-08-010710
+    - dir_perms_world_writable_system_owned_group
```

Comment:
You ensure that the testing stability files are updated. 

See the [tests](https://github.com/ComplianceAsCode/content/runs/5728468343?check_suite_focus=true#step:7:134) for details.